tests/freqai/test_freqai_datakitchen.py

```diff
@@ -194,3 +194,4 @@ def test_get_full_model_path(mocker, freqai_conf, model):
 
     model_path = freqai.dk.get_full_models_path(freqai_conf)
     assert model_path.is_dir() is True
+    shutil.rmtree(Path(freqai.dk.full_path))
```

Comment:
Not sure i understand this?

Assuming this is to delete some data this is creating - then this should be set to tmpdir (use as argument, it's a fixture in pytest) before the test / calls and set the base path to that- so it's only stored in the users temporary  directory (which is cleaned up automatically by pytest).

there's still another test that kinda leaves a `tensorflow` directory at the root (i know you're aware of that) - which suggests it's misconfigured to not use `user_data` (not necessarily the test - but the actual functionality), but current dir instead - which is kinda wrong.

Reply:
Tensorflow? We dont even have tensorflow in develop. I dont know which test you are referring to.

Reply:
yeah typo - tensorboard is the directory.

Reply:
I think I have resolved all lingering files.